.github/workflows/scxa-solrcloud.yaml

```
@@ -0,0 +1,47 @@
+name: Build and Push Image
+
+on:
+  push:
+    paths:
+      - "scxa-solrcloud/Dockerfile"
+
+jobs:
+  build:
+    name: Build and push image
+    runs-on: ubuntu-20.04
+    env:
+      IMAGE_NAME: scxa-solrcloud
+
+    if: github.ref == 'refs/heads/master'
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Read upstream tag without version
+      id: gettag
+      run: echo "::set-output name=tag::$(head -n 1 scxa-solrcloud/Dockerfile | awk -F':' '{print $2}' | awk -F'-' 'BEGIN { OFS="-" } {$NF=""; print $0}')"
+
+    - name: Read internal update version
+      id: getversion
+      run: echo "::set-output name=version::$(grep 'ARG RUNNER_VERSION' scxa-solrcloud/Dockerfile | awk -F'=' '{print $2}')"
+
+    - name: Build image
+      id: build-image
+      uses: redhat-actions/buildah-build@v2
+      with:
+        image: ${{ env.IMAGE_NAME }}
+        tags: latest ${{ steps.gettag.outputs.tag }}${{ steps.getversion.outputs.version }}
```

.. and remove the getversion reference here.

```suggestion
        tags: latest ${{ steps.gettag.outputs.tag }}
```